src/components/Header/Header.styles.ts

```diff
@@ -1,36 +1,64 @@
 import styled from "styled-components"
+import { Link } from "gatsby"
+import { breakpoints } from "../Shared"
 
 export const HeaderStyled = styled.header`
   max-width: 1440px;
   padding: 24px;
-  margin-bottom: "1.45rem";
+  margin-bottom: 1.45rem;
   display: flex;
-  background: "white";
-  color: #000;
+  align-items: center;
+  background: #fff;
   margin: 0 auto;
 `
 
-export const Nav = styled.div`
-  display: flex;
-`
+export const LinkStyled = styled(Link)`
+  width: 200px;
 
-export const H1 = styled.div``
+  @media ${breakpoints.md} {
+    width: 300px;
+    min-width: 200px;
+  }
+`
 
-export const NavLinks = styled.div`
-  margin-left: 48px;
+export const Nav = styled.nav<{ isOpen: boolean }>`
   display: flex;
   align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: ${({ isOpen }) => (isOpen ? "0" : "-100%")};
+  transition: right 0.3s ease;
+  background-color: #fff;
```

Comment:
Can you please add a `z-index` of at least a value of 5 to resolve this little overlap issue. :)

![image](https://user-images.githubusercontent.com/977413/95878693-169d3380-0da8-11eb-8ab2-f133b7f7049f.png)